lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
well, if the start of typing was the problem - then we would see an error in the video - which we don't either. We start typing too early, but still we have whoami > /dev/ttyS0 in the prompt